### PR TITLE
[STAN-174] UI: more spacing around search input

### PR DESCRIPTION
**Pull request by roc**
### Before
Screenshot 2022-01-13 at 11 55 20
Screenshot 2022-01-13 at 11 45 44
Screenshot 2022-01-12 at 15 57 11

### After
Screenshot 2022-01-12 at 15 57 42
Screenshot 2022-01-13 at 11 52 46
Screenshot 2022-01-13 at 11 45 47

### CODE REVIEW

File: ui/wdio.conf.js [hunk 1]
```diff
@@ -1,280 +1,276 @@
...
+  connectionRetryTimeout: 120000,
+  //
+  // Default request retries count
+  connectionRetryCount: 3,
+  //
+  // Test runner services
+  // Services take over a specific job you don't want to take care of. They enhance
+  // your test setup with almost no effort. Unlike plugins, they don't add new
+  // commands. Instead, they hook themselves up into the test process.
+  services: [['selenium-standalone', { drivers: { chrome: '95.0.4638.69' } }]],
```
Comment (lennym) [inline ui/wdio.conf.js:115]: Does this work with just the swap to selenium-standalone, but not pinning the version? Keeping the version fresh is probably preferable.

Comment (roc) [reply ui/wdio.conf.js:115]: I agree we want to be able to test against the edge of chrome, but I think we should be able to choose when we do that. At the moment integration tests will fail at the cadence of the chrome update cycle.  

I'm pinning the driver to the version we're setting in [package.json](https://github.com/nhsx/standards-registry/blob/STAN-174-search-squish/ui/package.json#L34) (95), and I think the service is a passthrough to [`wdio-chromedriver-service@7.2.2`](https://github.com/nhsx/standards-registry/blob/STAN-174-search-squish/ui/package.json#L37)

Comment (lennym) [reply ui/wdio.conf.js:115]: How come it's changed from `chromedriver` to `selenium-standalone`?

Comment (roc) [reply ui/wdio.conf.js]: Ah I understand your question a bit better now. 
```suggestion
  services: [['selenium-standalone']],
```
Let's try it out!

Comment (roc) [reply ui/wdio.conf.js]: That works. The default configuration of selenium-standalone is to take whatever's on the host (as far as I can tell) https://webdriver.io/docs/selenium-standalone-service/#configuration. The default configuration for chromedriver is to wrap a static version https://webdriver.io/docs/wdio-chromedriver-service

Comment (lennym) [reply ui/wdio.conf.js]: Probably want to swap out the chromedriver service for the selenium service in package.json

**Review by easternbloc — APPROVED** [review]

**Pull request merged**